triton/docker-compose.yml

```diff
@@ -0,0 +1,17 @@
+version: '2'
+
+services:
+
+    mysql:
+        extends:
+            file: ../common-compose.yml
+            service: mysql
+        network_mode: bridge
```

Comment:
Using an explicit `network_mode: bridge` here is what lets this `version: '2'` file still work on Triton (since no containers are using the `default` network, it doesn't get auto-created).